src/lib/parse_org.js

```diff
@@ -794,10 +778,21 @@ export const parseOrg = fileContents => {
       logBookEntries,
     } = parseDescriptionPrefixElements(header.get('rawDescription'));
 
+    const parsedDescription = parseRawText(strippedDescription);
+
+    const mergedPlanningItems = mergePlanningItems(
+      planningItems,
+      extractActiveTimestampsForPlanningItemsFromParse(
+        'TIMESTAMP_TITLE',
```

Comment:
Is there a good alternative to this `'STRING_CONSTANTS'`? I use them in multiple files and they are also used as values in hashs (the parse results).

I could define `export const STRING_CONSTANT = 'STRING_CONSTANT'` but that to be an overhead. I would always have to import them. And another developer might not know about the constant and just use the string literal then...

Reply:
That's a really good question. Some languages do offer good support for these "strings" that aren't really supposed to be strings, but some kind of identifier. Ruby calls them [symbols](https://ruby-doc.org/core-2.2.0/Symbol.html) (example: `:timestamp_title`), Clojure calls them [keywords](https://clojuredocs.org/clojure.core/keyword) (example: `timestamp_title`) - and both actually use the same syntax(;

JavaScript also has symbols and you can generate them with `Symbol()`. However, in JS a `Symbol` is always unique. Hence:

```
> Symbol('foo') != Symbol('foo')
true
```

You can use Symbols, if you want to, but then you'd have to import them everywhere like you said.

I'd say that many JS devs don't bother and use constant strings like you do. I'd opt for that, as well, and would add good jsdoc documentation for the functions parameters.